### DISCUSSION TITLE
LAB 07.02 Task 03

### DISCUSSION THREAD

**Discussion by joanfco30** [rramosp/ai4eng.v1#160] [07 - DISEÑO DE ALGORITMOS]
Buenas tardes, un cordial saludo. 
Quería hacer una observación en la función planteada en la tareas 3, por intuición cambie los valores de los array W0 a W2 y b0 a b2. 

Por último, agradecer por el desarrollo de este curso. Hoy lo termine, lleno todas mis espectativas y quede muy contento con el resultado. 

Hasta la próxima. 

![image](https://user-images.githubusercontent.com/60493083/177887444-dff155c2-066d-4bc6-a619-03fc273cfb01.png)

**Comment by Felipe-RA** [comment]
Gracias por tu feedback @joanfco30 

Nos alegra mucho que el curso haya cumplido tus expectativas

**Comment by rramosp** [comment]
gracias joanfco por la corrección.

nos alegra que el curso te haya gustado y ojalá te sea útil en tu vida
profesional.

compártelo con quien quieras

saludos
raúl

On Thu, Jul 7, 2022 at 6:27 PM joanfco30 ***@***.***> wrote:

> Buenas tardes, un cordial saludo.
> Quería hacer una observación en la función planteada en la tareas 3, por
> intuición cambie los valores de los array W0 a W2 y b0 a b2.
>
> Por último, agradecer por el desarrollo de este curso. Hoy lo termine,
> lleno todas mis espectativas y quede muy contento con el resultado.
>
> Hasta la próxima.
>
> [image: image]
> <https://user-images.githubusercontent.com/60493083/177887444-dff155c2-066d-4bc6-a619-03fc273cfb01.png>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/rramosp/ai4eng.v1/discussions/160>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADJRZIKDLHWAHJQRCZXXF5DVS5RUHANCNFSM527D7AFQ>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>

-- 
*Aviso legal:* El contenido de este mensaje y los archivos adjuntos son 
confidenciales y de uso exclusivo de la Universidad Nacional de Colombia. 
Se encuentran dirigidos sólo para el uso del destinatario al cual van 
enviados. La reproducción, lectura y/o copia se encuentran prohibidas a 
cualquier persona diferente a este y puede ser ilegal. Si usted lo ha 
recibido por error, infórmenos y elimínelo de su correo. Los Datos 
Personales serán tratados conforme a la Ley 1581 de 2012 y a nuestra 
Política de Datos Personales que podrá consultar en la página web 
www.unal.edu.co <http://www.unal.edu.co/>.* *Las opiniones, informaciones, 
conclusiones y cualquier otro tipo de dato contenido en este correo 
electrónico, no relacionados con la actividad de la Universidad Nacional de 
Colombia, se entenderá como personales y de ninguna manera son avaladas por 
la Universidad.
